planning/mapless_architecture/autoware_mission_lane_converter/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.8)
+project(autoware_mission_lane_converter)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# --- FIND DEPENDENCIES ---
+find_package(autoware_cmake REQUIRED)  # automatically find dependencies
+ament_auto_find_build_dependencies()
+autoware_package()
+
+# build executables
+ament_auto_add_executable(${PROJECT_NAME}
+  src/main_mission_lane_converter.cpp
+  src/mission_lane_converter_node.cpp)
+
+target_include_directories(${PROJECT_NAME} PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include>)
+
+target_compile_features(${PROJECT_NAME} PUBLIC c_std_99 cxx_std_17)  # Require C99 and C++17
+
+# install executable(s)
+install(TARGETS
+  ${PROJECT_NAME}
+  DESTINATION lib/${PROJECT_NAME})
+
+# install launchfile(s)/param file(s)
+install(DIRECTORY
+  launch
+  param
+  DESTINATION share/${PROJECT_NAME})
+
+# --- SPECIFY TESTS ---
+# configure clang format
+set(ament_cmake_clang_format_CONFIG_FILE ../../.clang-format)
+
+if(BUILD_TESTING)
+  find_package(ament_cmake_gtest REQUIRED)
+  find_package(ament_lint_auto REQUIRED)
+
+  ament_auto_add_gtest(${PROJECT_NAME}_tests test/gtest_main.cpp
+    test/src/test_mission_planner_converter.cpp
+    src/mission_lane_converter_node.cpp)
+
+  ament_lint_auto_find_test_dependencies()
+endif()
+
+ament_auto_package()
```

Comment:
`ament_auto_...` are not necessary. Please check other `CMakeLists.txt` files (e.g., in the `autoware_path_smoother`).

Reply:
Done!